### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by torarnv**

### CODE REVIEW

**Review by Swatinem — APPROVED** [review]
Wording changes look good, however I have no idea about the JS changes.

File: src/platforms/native/guides/minidumps/index.mdx [hunk 1]
```diff
@@ -10,15 +10,12 @@ description: "Learn about how Sentry processes Minidump crash reports."
...
 
-<Note>
 
-We strongly encourage you to use a higher-level SDK if possible, as Minidumps uploads are very limited in functionality:
+<Alert level="warning">
 
-- [_Native (C/C++)_](/platforms/native/)
-- [_Apple (Cocoa)_](/platforms/apple/)
-- [_Electron_](/platforms/electron/)
+This page describes **standalone** usage of Minidumps with Sentry. We strongly encourage you to use our [Native SDK](/platforms/native/), as Minidumps uploads are very limited in functionality.
```
Comment (jan-auer) [inline src/platforms/native/guides/minidumps/index.mdx:16]: Why would we no longer list the Apple and Electron SDKs here?

**Comment by torarnv** [comment]
Waiting for replies to earlier comments. Let me know if any changes are needed. I can push single change PRs if that helps.

File: src/platforms/native/guides/minidumps/index.mdx (hunk 1 above)
Comment (jan-auer) [reply src/platforms/native/guides/minidumps/index.mdx:16]: @torarnv Could you elaborate why Apple and Electron are no longer mentioned?

Comment (torarnv) [reply src/platforms/native/guides/minidumps/index.mdx:16]: I don't know if my comment wasn't posted or not. Does https://github.com/getsentry/sentry-docs/pull/2842#discussion_r553304455 clarify?

File: src/platforms/native/guides/breakpad/index.mdx [hunk 2]
```diff
@@ -8,13 +8,11 @@ description: "Learn about Sentry's integration with Google Breakpad."
 
 [Breakpad](https://chromium.googlesource.com/breakpad/breakpad/) is an open-source multiplatform crash reporting system written in C++ by Google and the predecessor of Crashpad. It supports macOS, Windows and Linux, and features an uploader to submit minidumps to a configured URL right when the process crashes.
 
-As opposed to Crashpad, Breakpad uses in-process crash reporting. This is less robust and has several disadvantages over out-of-process crash reporting. Unless you have integrated Breakpad already, we strongly recommend you to consider using Crashpad instead.
+<Alert level="warning">
```
Comment (PeloWriter) [inline src/platforms/native/guides/breakpad/index.mdx:11]: Please don't use Warning here, as the content doesn't rise to this level. Here's out guideline:
- **Note:** use these for useful context, information that should be visually called out from the core content. Use sparingly.
- **Important:** use these for information that's critical to know (this version introduces breaking changes). Use infrequently.
- **Warning:** use these for items that MUST be well understood before proceeding. These should be used very rarely.

File: src/platforms/native/guides/minidumps/index.mdx [hunk 3]
```diff
@@ -10,15 +10,12 @@ description: "Learn about how Sentry processes Minidump crash reports."
 Sentry can process Minidump crash reports, a memory dump used on Windows and by
 open-source libraries like [_Breakpad_](/platforms/native/guides/breakpad/) or [_Crashpad_](/platforms/native/guides/crashpad/).
 
-<Note>
 
-We strongly encourage you to use a higher-level SDK if possible, as Minidumps uploads are very limited in functionality:
+<Alert level="warning">
```
Comment (PeloWriter) [inline src/platforms/native/guides/minidumps/index.mdx:14]: Apply the same guideline:
- **Note:** use these for useful context, information that should be visually called out from the core content. Use sparingly.
- **Important:** use these for information that's critical to know (this version introduces breaking changes). Use infrequently.
- **Warning:** use these for items that MUST be well understood before proceeding. These should be used very rarely.

**Review by PeloWriter — COMMENTED** [review]
I didn't review the /component changes.

**Comment by torarnv** [comment]
Split into independent PRs for easier review:

 - https://github.com/getsentry/sentry-docs/pull/3367
 - https://github.com/getsentry/sentry-docs/pull/3368
 - https://github.com/getsentry/sentry-docs/pull/3369

File: src/platforms/native/guides/minidumps/index.mdx (hunk 3 above)
Comment (torarnv) [reply src/platforms/native/guides/minidumps/index.mdx:14]: Fixed in https://github.com/getsentry/sentry-docs/pull/3369

File: src/platforms/native/guides/breakpad/index.mdx (hunk 2 above)
Comment (torarnv) [reply src/platforms/native/guides/breakpad/index.mdx:11]: Fixed in https://github.com/getsentry/sentry-docs/pull/3369

**Pull request closed without merging**